### PR TITLE
Create ui.ko.ts

### PR DESCRIPTION
**Pull request by MarongHappy**

### CODE REVIEW

File: i18n/ui.ko.ts [hunk 1]
```diff
@@ -0,0 +1,815 @@
...
+    </message>
+    <message>
+        <location line="+55"/>
+        <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Welcome to AppImageLauncher!&lt;/span&gt;&lt;/p&gt;&lt;p&gt;This little helper is designed to improve your AppImage experience on your computer.&lt;/p&gt;&lt;p&gt;It appears you have never run AppImageLauncher before. Please take a minute and configure your preferences. You can always change these later on, using the control panel.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;AppImage 시작도구에 오신 것을 환영합니다!&lt;/span&gt;&lt;/p&gt;&lt;p&gt;이 작은 도우미는 컴퓨터에서 AppImage 환경을 개선하도록 설계되었습니다.&lt;/p&gt;&lt;p&gt;이전에 AppImage 시작도구를 실행한 적이 없는 것 같습니다. 잠시 시간을 내어 기본 설정을 구성하십시오. 나중에 제어판을 사용하여 언제든지 변경할 수 있습니다.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
+    </message>
+    <message>
+        <location line="+22"/>
+        <source>Ask me whether to move new AppImages into a central location</source>
+        <translation>새 AppImage를 중앙 위치로 이동할지 여부를 묻습니다</translation>
```
Comment (ernestwilhelmsen) [inline i18n/ui.ko.ts:19]: Exact copy-and-paste of Google Translate result. https://translate.google.com/?sl=en&tl=ko&text=Ask%20me%20whether%20to%20move%20new%20AppImages%20into%20a%20central%20location&op=translate

File: i18n/ui.ko.ts [hunk 2]
```diff
@@ -0,0 +1,815 @@
...
+        <location line="+13"/>
+        <location filename="../src/ui/update_main.cpp" line="+7"/>
+        <source>Not an AppImage: %1</source>
+        <translation>AppImage 아님: %1</translation>
+    </message>
+    <message>
+        <location line="+162"/>
+        <source>The directory the integrated AppImages are stored in is currently set to:
+%1</source>
+        <translation>통합 AppImages가 저장된 디렉터리는 현재 다음으로 설정되어 있습니다:
```
Comment (ernestwilhelmsen) [inline i18n/ui.ko.ts:334]: Why using "AppImages" here? Korean plural form is not as strict as English and using singular form will be suffice.

File: i18n/ui.ko.ts [hunk 3]
```diff
@@ -0,0 +1,815 @@
...
+    <message>
+        <location line="+1"/>
+        <source>Could not find update information in AppImage:
+%1
+
+The AppImage doesn&apos;t support updating. Please ask the authors to embed update information to allow for easy updating.</source>
+        <translation>AppImage에서 업데이트 정보를 찾을 수 없습니다:
+%1
+
+AppImage는 업데이트를 지원하지 않습니다. 쉽게 업데이트할 수 있도록 작성자에게 업데이트 정보를 포함하도록 요청하십시오.</translation>
```
Comment (ernestwilhelmsen) [inline i18n/ui.ko.ts:586]: Another exact copy of Google translation: https://translate.google.com/?sl=en&tl=ko&text=The%20AppImage%20doesn%27t%20support%20updating.%20Please%20ask%20the%20authors%20to%20embed%20update%20information%20to%20allow%20for%20easy%20updating.&op=translate

File: i18n/ui.ko.ts [hunk 4]
```diff
@@ -0,0 +1,815 @@
...
+    </message>
+    <message>
+        <location line="+5"/>
+        <source>Failed to remove old AppImage</source>
+        <translation>이전 AppImage를 제거하지 못했습니다</translation>
+    </message>
+    <message>
+        <location filename="../src/daemon/main.cpp" line="+101"/>
+        <source>Tracks AppImages in applications directories (user&apos;s, system and other ones). Automatically integrates AppImages moved into those directories and unintegrates ones removed from them.</source>
+        <translation>응용프로그램 디렉터리(사용자, 시스템 및 기타 디렉터리)에서 AppImages를 추적합니다. 해당 디렉터리로 이동된 AppImage를 자동으로 통합하고 제거된 디렉터리는 통합 해제합니다.</translation>
```
Comment (ernestwilhelmsen) [inline i18n/ui.ko.ts:644]: Another exact copy of Google translation: https://translate.google.com/?sl=en&tl=ko&text=Tracks%20AppImages%20in%20applications%20directories%20(user%27s%2C%20system%20and%20other%20ones).%20Automatically%20integrates%20AppImages%20moved%20into%20those%20directories%20and%20unintegrates%20ones%20removed%20from%20them.&op=translate Here, the thing unintegrated is not the directory as in your translation, but the AppImage. Google Translate translated that wrong.

File: i18n/ui.ko.ts [hunk 5]
```diff
@@ -0,0 +1,815 @@
...
+    </message>
+    <message>
+        <source>Search for AppImages in /Applications directories suitable mounted filesystems</source>
+        <translation type="vanished">마운트된 파일 시스템에 적합한 /Applications 디렉터리에서 AppImages 검색</translation>
+    </message>
+    <message>
+        <location line="+8"/>
+        <source>Lists directories watched by this daemon and exit</source>
+        <translation>재생된 디렉터리 목록
+이 데몬과 종료까지 감시된 디렉터리 목록</translation>
```
Comment (ernestwilhelmsen) [inline i18n/ui.ko.ts:658]: There is no "재생" involved in the source string.

File: i18n/ui.ko.ts [hunk 6]
```diff
@@ -0,0 +1,815 @@
...
+    </message>
+    <message>
+        <location line="+6"/>
+        <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When this box is checked, AppImageLauncher automatically starts a daemon called appimagelauncherd.&lt;/p&gt;&lt;p&gt;This daemon automatically integrates AppImages you copy into the &amp;quot;Applications directory&amp;quot; and the additional directories you configured. When the files are deleted, the daemon will clean up the integration data.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;이 상자를 선택하면 AppImage 시작도구가 appimagelauncherd라는 데몬을 자동으로 시작합니다.&lt;/p&gt;&lt;p&gt;이 데몬은 복사한 AppImage를 &amp;quot;응용프로그램 디렉터리&amp;quot;와 구성한 추가 디렉터리에 자동으로 통합합니다. 파일이 삭제되면 데몬이 통합 데이터를 정리합니다.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Auto start auto-integration daemon</source>
+        <translation>자동 시작 자동 통합 데몬</translation>
```
Comment (ernestwilhelmsen) [inline i18n/ui.ko.ts:733]: Another copy of Google translation: https://translate.google.com/?sl=en&tl=ko&text=Auto%20start%20auto-integration%20daemon&op=translate This option determins whether the daemon should be automatically started, so "자동 시작" should be go behind "자동 통합 데몬".

**Review by ernestwilhelmsen — COMMENTED** [review]
In general, I have found many of mistakes found by supposed direct copy of Google translation. I'd suggest not merging this translation as blindly using Google translate result may cause licensing and/or legal problems.

**Comment by TheAssassin** [comment]
@ernestwilhelmsen well I can't really tell about the quality of the translation, since I do not speak Korean. I generally also recommend people to use Weblate.

Would you like to fix the issues and provide translations using Weblate?

**Comment by MarongHappy** [comment]
OK.

2022년 5월 10일 (화) 오전 4:53, TheAssassin ***@***.***>님이 작성:

**Pull request closed without merging**